cypress/e2e/po/pages/extensions/kubewarden.po.ts

```diff
@@ -1,5 +1,5 @@
 import PagePo from '@/cypress/e2e/po/pages/page.po';
-import ExtensionsPagePo from '@/cypress/e2e/po/pages/extensions.po';
+// import ExtensionsPagePo from '@/cypress/e2e/po/pages/extensions.po';
```

Comment:
comment only. i'm really eager to get this in, the extension based tests have been real flaky recently and i think this PR will help. in a future commit though can you remove these commented out lines, or add a comment why they should stay?